src/main/scala/firrtl/proto/FromProto.scala

```diff
@@ -45,9 +45,9 @@ object FromProto {
       case Firrtl.SourceInfo.POSITION_FIELD_NUMBER =>
         val pos = info.getPosition
         val str = s"${pos.getFilename} ${pos.getLine}:${pos.getColumn}"
-        ir.FileInfo(ir.StringLit(str))
+        ir.FileInfo.fromUnescaped(str)
       case Firrtl.SourceInfo.TEXT_FIELD_NUMBER =>
-        ir.FileInfo(ir.StringLit(info.getText))
+        ir.FileInfo.fromUnescaped(info.getText)
```

Comment:
Ugh, we should stop escaping it in the proto, beyond the scope of this PR though because we obviously want to support old protos

Reply:
Yeah, I added this to make sure that the proto format stays the same.